src/utils/vc/helpers.js

```diff
@@ -1,12 +1,21 @@
 import { getPublicKeyFromKeyringPair } from '../misc';
 
+/**
+ * @typedef {object} KeyDoc The Options to use in the function createUser.
+ * @property {string} id TODOWRITE
+ * @property {any} controller TODOWRITE
+ * @property {any} type TODOWRITE
+ * @property {object} keypair TODOWRITE
+ * @property {object} publicKey TODOWRITE
+ */
+
 /**
  * Helper to get the key doc in a format needed for vc.js.
  * @param {string} did - DID in fully qualified form
  * @param {object} keypair - Keypair is generated by either using polkadot-js's keyring for Sr25519 and
  * Ed25519 or keypair generated with `generateEcdsaSecp256k1Keypair` for curve secp256k1.
  * @param {string} type - the type of key, Sr25519VerificationKey2020 or Ed25519VerificationKey2018 or EcdsaSecp256k1VerificationKey2019
- * @returns {{publicKeyBase58: *, controller: *, id: string, type: *, privateKeyBase58: (string|KeyObject|T2|Buffer|CryptoKey)}}
+ * @returns {KeyDoc}
```

Comment:
`KeyDoc` -> `function`

Reply:
there is a typedef for KeyDoc https://github.com/docknetwork/sdk/pull/91/files/883a500eb396263547fd40b5e865bfca2e264a00#diff-cbda9485de00c1a8f3f71fac0d2ed188